tests/e2e/index.spec.ts

```diff
@@ -82,6 +84,14 @@ test.describe("Index page", () => {
         await expect(await firstRegion.getAttribute("stroke")).not.toEqual(stroke);
     });
 
+    test("country outline is shown", async ({ page }) => {
+        const firstRegion = await getNthRegion(page, 1);
+        await firstRegion.click();
+        await page.waitForURL(/dengue\/may24\/FOI\/AGO/i);
+        await expect(await page.locator("div.spinner")).toHaveCount(0);
+        await expect(await page.locator(COUNTRY_OUTLINE_SELECTOR)).toHaveCount(1);
+    });
```

Comment:
This might add extra confidence (that the country outline is only shown when needed):

```suggestion
    test("country outline is shown", async ({ page }) => {
        const firstRegion = await getNthRegion(page, 1);
        // assuming page is fully loaded
        await expect(await page.locator(COUNTRY_OUTLINE_SELECTOR)).toHaveCount(0);
        await firstRegion.click();
        await page.waitForURL(/dengue\/may24\/FOI\/AGO/i);
        await expect(await page.locator("div.spinner")).toHaveCount(0);
        await expect(await page.locator(COUNTRY_OUTLINE_SELECTOR)).toHaveCount(1);
    });
```

Reply:
yh nice!

Reply:
I shouldn't have used the `suggest` function here, because the line `// assuming page is fully loaded` is just a placeholder for somebody actually checking that the page has loaded by this point in the test. (Or we risk false positives)

Reply:
ohhh okay got you, i can fill that in!